packages/tame-metering/src/tame.js

```diff
@@ -37,8 +46,8 @@ export function tameMetering() {
   const FunctionPrototype = Function.prototype;
 
   let globalMeter = null;
-  const definePropertyFailures = new Set();
-  const frozenPaths = new Set();
+  const definePropertyFailures = new Set(DEFINE_PROPERTY_FAILURES_ALLOWLIST);
+  const frozenPaths = new Set(FROZEN_PATHS_ALLOWLIST);
```

Comment:
interesting... so we had an extensibility feature that we weren't using, and it just so happens to fit our current need?

Reply:
It used to prevent logging the same error more than once (mutable set that we update).  Prepopulating it maks things quieter.